src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -27,6 +27,12 @@ public class MicrobeEditor : Node, ILoadableGameState, IGodotEarlyNodeResolve
     [JsonProperty]
     public float CurrentOrganelleCost;
 
+    private const float ARROW_OFFSET = 3.5f;
+    private const float HEX_HEIGHT = 1.3f;
```

Comment:
This could probably be confused with or relating to `DEFAULT_HEX_SIZE` constant: https://github.com/Revolutionary-Games/Thrive/blob/18e8bf7e9e3294d09b493089d5ac4994e28f4584/simulation_parameters/Constants.cs#L24 A short description of the variable's purpose should probably suffice.

Reply:
Should HEX_HEIGHT be calculated as a derivate value from DEFAULT_HEX_SIZE? It might be related.
Not saying we'll ever change the DEFAULT_HEX_SIZE, but just in case the derived value should be computed from it.

Reply:
Probably extra work if `DEFAULT_HEX_SIZE` is rarely changed, but that makes sense I guess since it seems related indeed.